update_baselines.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
Was it intentional to check this in? It seems quite useful.

Reply:
yes, and it was very easy :-)